appendices/versions.adoc

```diff
@@ -206,6 +206,9 @@ via flink:vkGetPhysicalDeviceFeatures2.
 
 === Differences relative to `VK_EXT_shader_viewport_index_layer`
 
+The code:ShaderViewportIndexLayerEXT SPIR-V capability was replaced the with
+the code:ShaderViewportIndex and code:ShaderLayer capability. Declaring both
```

Comment:
'the with the'

Otherwise functionally correct

Reply:
Also 'capabilities' at the end of sentence, not 'capability'.

Reply:
```suggestion
the code:ShaderViewportIndex and code:ShaderLayer capabilities. Declaring both
```